### PR TITLE
feat: add SKIP_TLS_VALIDATION flag to client

### PR DESCRIPTION
**Pull request by palicena**
This adds a flag that allows us to skip TLS validation when the client makes https requests. It currently modifies the default transport, which is not ideal, but it's a quick and neat-ish solution for the moment. It's for development purposes only, of course.

### CODE REVIEW

File: pkg/bridgemodel/utils.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (t *HttpClient) sendAuthorizedRequest(method string, url string, body []byt
 	return
 }
 
-func ConfigureDefaultTransport() {
+func ConfigureDefaultTransportToSkipTlsValidation() {
```
Comment (palicena) [inline pkg/bridgemodel/utils.go:71]: This function wasn't actually used so I renamed it and made use of it since it did exactly what I needed it to do.

**Review by raghu-sarangapani — APPROVED** [review]

**Review by jharrod — APPROVED** [review]

File: k8sconfig/client.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ data:
   cloud_bridge_url: "http://message-bridge.hcieng.com"
   key_store: "file:///data"
   log_leve: "trace"
+  skip_tls_validation: "true"
```
Comment (jharrod) [inline k8sconfig/client.yaml:11]: should this be defaulted to false?

Comment (palicena) [reply k8sconfig/client.yaml:11]: correct!

File: pkg/bridgeclient/app.go [hunk 3]
```diff
@@ -72,6 +72,12 @@ func RunClient(test bool) {
 	if msgFormat == nil {
 		log.Fatalf("Unable to get the message format")
 	}
+
+	if pkg.Config.SkipTlsValidation {
+		log.Warn("SKIP_TLS_VALIDATION was set to true! Don't use this in production!")
+		bridgemodel.ConfigureDefaultTransportToSkipTlsValidation()
```
Comment (bmason42) [inline pkg/bridgeclient/app.go:78]: are you sure the function name is descriptive enough???

**Review by bmason42 — APPROVED** [review]

File: pkg/bridgeclient/app.go (hunk 3 above)
Comment (palicena) [reply pkg/bridgeclient/app.go:78]: I really didn't want anyone to confuse it for something else 😅

**Pull request merged**